src/main.rs

```diff
@@ -199,22 +212,29 @@ async fn hello_world() -> Html<&'static str> {
 
 #[derive(Clone)]
 pub struct AppConfig {
-    github_max_creation_time: DateTime<FixedOffset>,
-    eth_check_nonce_at_block: String,
-    eth_min_nonce:            i64,
-    eth_rpc_url:              String,
+    github_max_creation_time:    DateTime<FixedOffset>,
+    eth_check_nonce_at_block:    String,
+    eth_min_nonce:               i64,
+    eth_rpc_url:                 String,
+    transcript_file:             PathBuf,
+    transcript_in_progress_file: PathBuf,
 }
```

Comment:
This struct should be merged into `crate::main::Options`. (And appropriate `#[clap(...)]` and docstring attributes added).